R/teal_card.R

```diff
@@ -50,7 +50,8 @@ teal_card <- function(...) {
 #' @export
 #' @keywords internal
 teal_card.default <- function(...) {
-  x <- list(...)
+  x <- sapply(list(...), .convert_teal_card_input, simplify = FALSE, USE.NAMES = TRUE)
```

Comment:
`lapply` preserves the names by default if `x` is a named list.
```suggestion
  x <- lapply(list(...), .convert_teal_card_input)
```